### PR TITLE
Aaron Schreck's Scrabble Extravaganza 

### PR DESCRIPTION
**Pull request by aschreck**

### CODE REVIEW

File: lib/scrabble.rb [hunk 1]
```diff
@@ -1,7 +1,36 @@
 class Scrabble
 
+  def standardized_format(word)
```
Comment (VictoriaVasys) [inline lib/scrabble.rb:3]: I like that you broke out this method! Just make sure you're testing for any methods you create.

File: lib/scrabble.rb [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+    word.upcase
+  end
+
   def score(word)
-    1
+    return 0 if word.nil?
+
+    values = point_values
```
Comment (VictoriaVasys) [inline lib/scrabble.rb:10]: You can just call point_values wherever it's needed instead of assigning it to a variable here.

File: lib/scrabble.rb [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
+    word = standardized_format(word)
+
+    sum = 0
+    word.each_char do |char|
```
Comment (VictoriaVasys) [inline lib/scrabble.rb:14]: We'd love to see you pushing to use more advanced enums, like reduce here. It's not expected quite yet, but the sooner you practice, the better you'll become.

File: lib/scrabble.rb [hunk 4]
```diff
@@ -1,7 +1,36 @@
...
+      sum += values[char]
+    end
+    sum
+  end
+
+  def score_with_multipliers(word, board_array, multiplier = 1)
```
Comment (VictoriaVasys) [inline lib/scrabble.rb:20]: Think about ways that you could break the logic in this method into different parts (new methods).

File: test/scrabble_test.rb [hunk 5]
```diff
@@ -6,8 +6,37 @@
...
   end
+
+  def test_it_can_test_a_word
+    
+    assert_equal 3, Scrabble.new.score("ant")
+    assert_equal 11,  Scrabble.new.score("doggeral")
+  end
+
+  def test_it_handles_nil_and_empty
+
```
Comment (VictoriaVasys) [inline test/scrabble_test.rb:21]: Be aware of spacing in your tests (only space between setup & assertions).

**Review by VictoriaVasys — COMMENTED** [review]
Great job overall! We love to see that you committed often, your functionality is solid and you started to make an effort to break out methods.  Continue to work on that, make sure to test every method you create & test for edge cases.
